scripts/tabulate/asset_pricing_tab.py

```diff
@@ -32,11 +32,11 @@
             print(f"Processing {panel_info} {dominance} {frequency}")
 
             df_ap = (
-                asset_pricing(df_panel, dominance, 3, frequency, True)
+                asset_pricing(df_panel, [0.8], dominance, frequency, False)
                 .set_index("Portfolios")
                 .T
                 if dominance != "ret"
-                else asset_pricing(df_panel, dominance, 3, frequency, False)
+                else asset_pricing(df_panel, [0.8], dominance, frequency, False)
```

Comment:
i don't understand the difference between the two statements before and after `if`